### PR TITLE
3 get categories

### PR DESCRIPTION
**Pull request by ElenaCKay**

### CODE REVIEW

File: controllers/categories.controller.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const { fetchCategories } = require("../models/categories.model");
+
+exports.getCategories = (req, res) => {
+    fetchCategories().then((categoryArray) => {
+        res.status(200).send({ categories: categoryArray });
+    });
+};
```
Comment (Hard-Luck) [inline controllers/categories.controller.js:11]: Would need a catch block on this endpoint and an extra parameter in line 3

File: app.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+const { getCategories } = require("./controllers/categories.controller");
+
+app.get("/api", (req, res) => {
+    res.status(200).send({ msg: "Server is up and running" });
+});
+
+app.get("/api/categories", getCategories);
+
```
Comment (Hard-Luck) [inline app.js:10]: We would need an endpoint that catches all endpoints not specified and responds with a 404: Not found

**Review by Hard-Luck — CHANGES_REQUESTED** [review]
A couple of small changes and then a resubmit please, along with these changes in the comments we should test that the 404 endpoint works, and we should have a error handling middleware that catches uncaught errors and responds with a 500.

**Review by HeatherEver — APPROVED** [review]
Lovely changes ⭐

**Pull request merged**